CHANGELOG.md

```diff
@@ -20,12 +20,15 @@ Here is a template for new release sections
 ```
 ## [Unreleased]
 
-
-## [0.0.7] 2019-08-29
-
 ### Added
 - CHANGELOG
 
+### Changed
+- updated link to the examples/tutorials ([#24](https://github.com/OpenEnergyPlatform/oedialect/pull/24)
+
+### Removed
+- notebook example ([#24](https://github.com/OpenEnergyPlatform/oedialect/pull/24)
```

Comment:
dito

Reply:
Fixed in b301e656cf4a0a4e2ce87fe05480e645c9f69253